jerry-core/lit/lit-magic-strings.inc.h

```diff
@@ -53,6 +53,7 @@ LIT_MAGIC_STRING_DEF (LIT_MAGIC_STRING_LN2_U, "LN2")
 LIT_MAGIC_STRING_DEF (LIT_MAGIC_STRING_NAN, "NaN")
 LIT_MAGIC_STRING_DEF (LIT_MAGIC_STRING_UTC_U, "UTC")
 LIT_MAGIC_STRING_DEF (LIT_MAGIC_STRING_ABS, "abs")
+LIT_MAGIC_STRING_DEF (LIT_MAGIC_STRING_ALL, "all")
```

Comment:
With #1690 , the definition of magic strings changed. Please, rebase the commit onto current master and add these new strings to `jerry-core/lit/lit-magic-strings.ini`. Then, by running `tools/gen-magic-strings.py`, you'll re-generate this `.inc.h` file. (Should you accidentally forget to do that, `tools/run-tests.py` will remind you anyway.)